src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderContentTypes.java

```diff
@@ -68,6 +72,8 @@
             new StringHierarchicalField(Arrays.asList("content"), "component", FieldContentIdentifier.CATEGORY_ITEM, LABEL_COMPONENT_NAME + HierarchicalField.LABEL_URL_SUFFIX),
             new StringHierarchicalField(Arrays.asList("content"), "componentVersionName", FieldContentIdentifier.CATEGORY_ITEM, LABEL_COMPONENT_VERSION_NAME),
             new StringHierarchicalField(Arrays.asList("content"), "componentVersion", FieldContentIdentifier.CATEGORY_ITEM, LABEL_COMPONENT_VERSION_NAME + HierarchicalField.LABEL_URL_SUFFIX),
+            new StringHierarchicalField(Arrays.asList("content"), "firstName", FieldContentIdentifier.CATEGORY_ITEM, LABEL_POLICY_OVERRIDE_FIRST_NAME),
+            new StringHierarchicalField(Arrays.asList("content"), "lastName", FieldContentIdentifier.CATEGORY_ITEM, LABEL_POLICY_OVERRIDE_LAST_NAME),
```

Comment:
Why do we have the static strings above and we aren't using them here?

Reply:
I'll refactor what is reused.